docs/aws/README.md

```diff
@@ -1,11 +1,40 @@
-# AWS
+# Getting Started with Karpenter on AWS
 
-This guide will provide a complete Karpenter installation for AWS.
-These steps are opinionated and may need to be adapted for your use case.
+Karpenter automatically provisions new nodes in response to unschedulable
+pods. Karpenter does this by observing events within the Kubernetes cluster,
+and then sending commands to the underlying cloud provider. 
 
-> This guide should take less than 1 hour to complete and cost less than $.25
+In this example, the cluster is running on Amazon Web Services (AWS) Elastic
+Kubernetes Service (EKS). Karpenter is designed to be cloud provider agnostic,
+but currently only supports AWS. Contributions are welcomed. 
+
+This guide should take less than 1 hour to complete, and cost less than $0.25.
+Follow the clean-up instructions to reduce any charges.
+
+# Install
+
+Karpenter is installed in clusters with a helm chart.
+
+Karpenter additionally requires IAM Roles for Service Accounts (IRSA). IRSA
```

Comment:
I'd be more crisp that Karpenter doesn't necessarily require IRSA, but it does require IAM roles. IRSA is a convenient way to manage IAM roles when using Kubernetes.

Also, IMO "as the cloud provider" is superflous.